spec/factories/recipients.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :recipient do
+    association :notification, factory: :notification
```

Comment:
I would prefer not to have the associations in the Factories. You can always create them as needed in the specs. However, when they're in the factory, you loose that control. 
Let's say you want to check that a validation on recipient is good; perhaps response_time can't be in the future, say. So you have the factory create one for you. Because of the associations, it will also create a notification and a person. And hey, notification factory lists an association to an Event ! Now you've created four objects to test one validation. 

That make sense ?
Thanks for your help,
Kevin